fastlane/Fastfile

```diff
@@ -43,7 +43,7 @@ platform :ios do
     # Need to run scan multiple times
     scan(
       step_name: "scan - iPhone", 
-      devices: ["iPhone 6", "iPhone 12"]
+      devices: ["iPhone 8", "iPhone 12"]
```

Comment:
replaced because I was still getting 
<img width="607" alt="Screen Shot 2021-09-16 at 6 04 56 PM" src="https://user-images.githubusercontent.com/3922667/133686893-7c0150f3-663b-4d24-b80f-74e454645532.png">
but the iPhone 8 works correctly